app/queries/v2/trade_lead/query.rb

```diff
@@ -13,6 +13,7 @@ def initialize(options = {})
       @publish_date = options[:publish_date] if options[:publish_date].present?
       @end_date = options[:end_date] if options[:end_date].present?
       @publish_date_amended = options[:publish_date_amended] if options[:publish_date_amended].present?
+      @categories  = options[:categories].split(',').map(&:strip) rescue nil
```

Comment:
Even though you'll see `var = statement rescue nil` here and there in the codebase, it's [not good to do so](https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers). Better to do `var = statement if something`.